docs/guide/intro/index.adoc

```diff
@@ -397,7 +397,7 @@ For example such a script would create an account *admin/passW0rd!*
 /subsystem=elytron/security-domain=ManagementDomain:list-remove(name=realms, index=0)
 /subsystem=elytron/properties-realm=ManagementRealm:remove
 /subsystem=elytron/filesystem-realm=ManagementRealm:add(path=management-realm, relative-to=jboss.server.config.dir)
-reload
+reload --start-mode=admin-only
```

Comment:
I am wandering if this reload is really required. Would that work for you to simply remove the reload? I feel that it is a typo.

Reply:
No, it does not work without the `reload` command. If I remove it then adding the identy (`...:add-identy(identity=admin)`) fails with the error message: "WFLYCTL0158: Operation handler failed: org.jboss.msc.service.ServiceNotFoundException: service org.wildfly.security.modifiable-security-realm.ManagementRealm not found".

Reply:
@cboehme , thank-you for your testing. Your changes look good then. Thank-you.